DalamudPluginProjectTemplateFSharp.fsproj

```diff
@@ -6,23 +6,28 @@
 		<PlatformTarget>x64</PlatformTarget>
 		<ProduceReferenceAssembly>false</ProduceReferenceAssembly>
 		<AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+		<CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
```

Comment:
Needed to copy `FSharp.Core.dll` to output, otherwise Dalamud can't load the assembly.